routes/user-api.js

```diff
@@ -5,10 +5,10 @@
 
 //init the required modules
 var express = require('express');
-var log = require('debug-logger')('user-api.js');
-var validator = require('validator');
-var crypto = require('crypto');
+var log = require('debug-logger')('user-api.js');;
```

Comment:
Fixed in commit https://github.com/deleepa/simple-api-server/pull/25/commits/5858bc0da061efd3a1412ae08aa4e8172a3b2e0f of the same PR